en/server-side/nodejs.md

```diff
@@ -51,7 +65,13 @@ Hello World!
 
 Express is a popular web framework for Node.js. It provides a simple and elegant API for building web applications. Let's use Express to create a simple web server that will respond to HTTP requests with a "Hello World!" message.
 
-First, we need to install Express. To do this, run the following command in your terminal:
+But before we install Express, we must first run this in order to set up npm, javascript's package manager:
```

Comment:
What do you think about this pharse?

First, we need to initialize npm by running `npm init` and then install Express.

Reply:
I think it's good

Reply:
Is it possible to update the PR with that content?